### PR TITLE
Added a lottie animation loader to the website

### PR DESCRIPTION
**Pull request by PulkitShubham**
---

## Type of PR
- **Add X in the box to specify the improvement type.**
- [ ] Bug fix
- [X] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): ___________

## Description
This pull request aims to implement a Lottie animation loader that displays while the main content of the webpage is loading. The loader is centered on the screen and fades out smoothly once the page is fully loaded, enhancing the user experience during loading times.

## Screenshots / Videos (if applicable)
**Before:**
- The webpage displayed no loader during loading, leading to a blank screen that might confuse users.

**After:**
- A visually appealing Lottie loader appears at the center of the screen while the content is loading, providing a clear indication that the page is in the process of loading.

image

## Checklist
- **Add X in the box to specify.**
- [X] I have performed a self-review of my code.
- [X] I have tested the changes thoroughly before submitting this pull request.
- [X] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [X] I have commented my code, particularly in hard-to-understand areas.

### Additional Context
The loader was created using the Lottie web library, which allows for scalable vector animations. The animation URL used is a publicly available example. The implementation includes a fade-in effect for the main content and prevents scrolling while the loader is visible. No additional issues were encountered during the implementation.

Thank you for reviewing my pull request!

---

### CODE REVIEW

**Comment by PulkitShubham** [comment]
@swaraj-das Kindly review my PR I've worked quite hard to get an animated loader issue no #540

File: index.html [hunk 1]
```diff
@@ -25,10 +25,58 @@
 
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/swiper@11/swiper-bundle.min.css" />
     <link rel="stylesheet" href="./popup/popup.css" />
+    <style>
```
Comment (swaraj-das) [inline index.html:28]: I can't merge if you add css  and js in html file. Please write css inside the style.css and js inside script.js

Comment (PulkitShubham) [reply index.html:28]: Okayyy

**Comment by PulkitShubham** [comment]
@swaraj-das I have rectified the issue and added Loader.js and Loader.css files

**Review by swaraj-das — APPROVED** [review]

**Pull request merged**